tests/scf-hess2/input.dat

```diff
@@ -0,0 +1,89 @@
+#! UHF STO-3G (Cartesian) and cc-pVDZ (spherical) water Hessian test, against Psi3 reference values.
+#! This test should match RHF values exactly
+
+psi3_hess = np.array([                                                                                    #TEST
```

Comment:
be sure to add the new tests to `tests/CMakeLists.txt` so they run.

Reply:
Ah yes, thanks!